### ISSUE TITLE
PID Min on, Max on, and Max off settings not saving

### ISSUE THREAD

**Issue by BirdBare** [kizniche/Mycodo#810]
### Describe the problem/bug

My PID setup: I am running fans on an on or off output. No PWM
What am I trying to accomplish. I'm using Mycodo to control temp inside of a charcoal grill for smoking.
What is the Issue: In order to get better temp control, I need the output to turn on a minimum of 10 seconds and a max of 50 seconds each period(60 sec). I've tried to set the values in the PID controller but they are not saved. 

### Versions:

Latest version of everything

### Reproducibility

Go into PID controller and try to set and save the Min off, Max on, Min on values with the controller activated or deactivated.

<img width="1139" alt="Screen Shot 2020-08-08 at 4 12 23 PM" src="https://user-images.githubusercontent.com/1051490/89719844-0e80dc00-d992-11ea-88fd-ffe624a5d532.png">

<img width="548" alt="Screen Shot 2020-08-08 at 4 13 26 PM" src="https://user-images.githubusercontent.com/1051490/89719855-1d678e80-d992-11ea-8416-5a3285207509.png">

<img width="1147" alt="Screen Shot 2020-08-08 at 4 13 50 PM" src="https://user-images.githubusercontent.com/1051490/89719867-29535080-d992-11ea-99ce-1199f051eaf6.png">

Log line: 2020-08-08 16:12:33,883 - INFO - mycodo.controllers.controller_pid_a3882021 - PID Settings: Device ID: f071c9db-b324-495c-aa2e-f35ae44edf6c, Measurement ID: 8ec92364-26a7-48e8-ab09-192dc68e28c4, Direction: raise, Period: 60.0, Setpoint: 225.0, Band: 0.0, Kp: 15.0, Ki: 0.0, Kd: 0.0, Integrator Min: -100.0, Integrator Max 100.0, Output Raise: ae525a35-5aac-4fd8-8e5b-f5ece9c49cc2, Output Raise Type: on_off, Output Raise Min On: 0.0, Output Raise Max On: 0.0, Output Raise Min Off: 0.0, Output Raise Always Min: False, Output Lower: None, Output Lower Type: on_off, Output Lower Min On: 0.0, Output Lower Max On: 0.0, Output Lower Min Off: 0.0, Output Lower Always Min: False, Setpoint Tracking Type: , Setpoint Tracking ID:

Basically, for some reason the PID values are being ignored.

Thanks!

**Comment by kizniche** [comment]
Duplicate of #805
